setup.py

```diff
@@ -26,7 +26,7 @@ def get_version(rel_path):
     "click>=7.0,<9.0",
     "cfn-flip>=1.2.3,<2.0",
     "deepdiff>=5.5.0,<6.0",
-    "PyYaml>=5.1,<6.0",
+    "PyYaml>=5.1,<5.3",
```

Comment:
I'm curious why you didn't make it PyYam > 6.0. They made a fix for that there.

Reply:
ahh, i didn't realize that.  i'll try `PyYam > 6.0` and see if it's still green.